src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/CommonTestResourcesAccess.kt

```diff
@@ -0,0 +1,25 @@
+package it.krzeminski.snakeyaml.engine.kmp
+
+import CommonTestResources
+import okio.ByteString
+
+/**
+ * Retrieves a string content from common resources using a given path.
+ * TODO: Add proper error handling if the resource doesn't exist.
+ *
+ * @param path The slash-delimited path to the resource.
+ * @return The content of the resource as a UTF-8 encoded string with normalized line breaks.
+ */
+fun stringFromResources(path: String): String {
+    val segments = path.split("/")
+    return (segments
+        .dropLast(1)
+        .fold(CommonTestResources.resourcesMap) { map, segment ->
+            @Suppress("UNCHECKED_CAST")
+            map[segment] as Map<String, Any>
```

Comment:
issue(non-blocking): this part will throw NPE if there is no segment in the map. We probably should use `requireNotNull` or `checkNotNull` with error message e.g. "resource {name} does not exist"